src/index.js

```diff
@@ -1,34 +1,51 @@
+import PropTypes from 'prop-types';
 import React from 'react';
 import 'react-app-polyfill/ie11'; // For IE 11 support
 import ReactDOM from 'react-dom';
 import { OverridableContext } from 'react-overridable';
-import { Link } from 'react-router-dom';
 import 'semantic-ui-less/semantic.less';
-import { Item } from 'semantic-ui-react';
 import { InvenioILSApp } from './lib';
+import 'semantic-ui-less/semantic.less';
```

Comment:
double import? Isn't it already above?

Reply:
I didn't notice it in first place, it got all cleaned up in the upcoming PR #58